src/App.js

```diff
@@ -1,25 +1,11 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import ScholarshipListPage from './pages/ScholarshipListPage';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <ScholarshipListPage />
```

Comment:
Since we'll probably have a separate home page, could we instead use [React Router](https://reactrouter.com/web/guides/quick-start) to only to route to the Scholarships page if the path matches `/scholarships` or so?